### PR TITLE
Disconnect callback

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -140,20 +140,25 @@ type Client interface {
...
-	cache      map[string]map[string]libovsdb.Row
-	cachemutex sync.RWMutex
-	tranmutex  sync.Mutex
-	callback   OVNSignal
+	client       *libovsdb.OvsdbClient
+	cache        map[string]map[string]libovsdb.Row
+	cachemutex   sync.RWMutex
+	tranmutex    sync.Mutex
+	callback     OVNSignal
+	disconnected OVNDisconnectedCallback
```
Comment (vtolstov) [inline client.go]: may be rename this like disconnectCB and callback to signalCB ?

File: client.go [hunk 2]
```diff
@@ -140,20 +140,25 @@ type Client interface {
...
 }
 
 func NewClient(cfg *Config) (Client, error) {
 	imp := &ovndb{
-		cache:    make(map[string]map[string]libovsdb.Row),
-		callback: cfg.SignalCB,
+		cache:        make(map[string]map[string]libovsdb.Row),
+		callback:     cfg.SignalCB,
+		disconnected: cfg.DisconnectCB,
```
Comment (vtolstov) [inline client.go]: rename like signalCB and disconnectCB?

File: client.go (hunk 1 above)
Comment (lebauce) [reply client.go]: Indeed. Done

File: client.go (hunk 2 above)
Comment (lebauce) [reply client.go]: done

**Comment by vtolstov** [comment]
thanks!

**Comment by lebauce** [comment]
Thank you for your quick review !

**Pull request merged**